ui/app/AppLayouts/Chat/controls/community/PermissionsDropdown.qml

```diff
@@ -114,20 +114,14 @@ StatusDropdown {
 
         // TODO: Channel-level permissions are temporarily hidden until they are
```

Comment:
Does this TODO still apply now what you uncommented the section below?

Reply:
This comment can be removed indeed